src/phoenix/logging/_filter.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+
+class NonErrorFilter(logging.Filter):
+    # @override
```

Comment:
```suggestion
```

Reply:
I left them intentionally to add them as we drop support for older python, can remove